examples/40_paper/2018_neurips_perrone_example.py

```diff
@@ -197,12 +197,12 @@ def list_categorical_attributes(flow_type='svm'):
 # Selecting a task for the surrogate
 task_id = task_ids[-1]
 print("Task ID : ", task_id)
-X, y = create_table_from_evaluations(eval_df, task_ids=[task_id], flow_type='svm')
+features, labels = create_table_from_evaluations(eval_df, task_ids=[task_id], flow_type='svm')
 
-model.fit(X, y)
-y_pred = model.predict(X)
+model.fit(features, labels)
+y_pred = model.predict(features)
```

Comment:
Could you please rename variables such as `y_pred`, too, for example to `predictions`?